packages/react/react/src/hooks/usePreservedReference.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { renderHook } from '@testing-library/react';
+import { useEffect, useState } from 'react';
+import { useCallbackOnce } from './useCallbackOnce';
+import { usePreservedReference } from './usePreservedReference';
+
+type TossObject = {
+  toss: string;
+};
+
+describe('usePreservedReference', () => {
+  const callbackReturnTrue = jest.fn(() => true);
+  const callbackReturnFalse = jest.fn(() => false);
+  const tossObject: TossObject = { toss: '토스' };
+
+  const useTestCustomHook = (object: TossObject, areValuesEqual?: () => boolean) => {
+    const [reference, setReference] = useState<TossObject>(tossObject);
+    const ref = usePreservedReference<TossObject>(reference, areValuesEqual);
+
+    // 마운트 시 한번만 실행되는 코드
```

Comment:
It's not that complicated. I think the hook name shows the role well
```suggestion
```
Is there a reason you left an annotation?
If you need to leave an comment, you'd better leave it in English according to the convention😄